conductr_cli/conduct.py

```diff
@@ -109,24 +110,26 @@ def build_parser():
     # Sub-parser for `events` sub-command
     events_parser = subparsers.add_parser('events',
                                           help='show bundle events')
-    events_parser.add_argument('--service',
-                               default='http://{}:9210'.format(default_ip),
-                               help='Events service address')
+    add_ip_and_port(events_parser)
+    events_parser.add_argument('--count',
```

Comment:
@huntc - I have introduced the `--count` switch to indicate how many events/logs to be fetched. Please let me know if you think this is ok.